packages/select/src/vaadin-lit-select-item.js

```diff
@@ -35,6 +35,23 @@ class SelectItem extends ItemMixin(ThemableMixin(DirMixin(PolylitMixin(LitElemen
     `;
   }
 
+  static get properties() {
+    return {
+      /**
+       * Use property instead of setting an attribute in `ready()`
+       * to ensure that `role` can be removed from the item clone,
+       * which is done synchronously after the clone is attached.
+       *
+       * @protected
```

Comment:
nit: not sure if this can be marked protected https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals/role

Reply:
Thanks, I wasn't aware of this API. I think we can keep `@protected` to hide property from the API docs.

Reply:
Ah, good point 👍